### ISSUE TITLE
Improvements

### ISSUE THREAD

**Issue by DiegoSousa123** [nikita36078/J2ME-Loader#355]
It could be added the possibility of editing a control layout in recessed mode and another in landscape mode.

**Comment by 1234Sameer** [comment]
You mean portrait and landscape layouts Independent of each other right ? Yeah, that would be great. Right now editing portrait layout also affects landscape layout.

**Comment by BeF1990** [comment]
Yes, that's very good idea. But will be really great if I could set different control layout for each game. Sometimes is very useful.

**Comment by 1234Sameer** [comment]
@BeF1990 
Use the template feature in the app.

**Comment by BeF1990** [comment]
Oh sorry. Of course it's working already.

**Comment by oatmeal01** [comment]
Templates is the solution to this problem. Update to the latest version.
